src/libm-tester/CMakeLists.txt

```diff
@@ -52,13 +52,38 @@ macro(test_extension SIMD)
       endif (COMPILER_SUPPORTS_WEAK_ALIASES)
     endif(ENABLE_GNUABI)
 
+    # Build tester2 SIMD
+    string(TOLOWER ${SIMD} SCSIMD)
+    foreach(P dp sp)
+      set(T "tester2${SCSIMD}${P}")
+      add_executable(${T} tester2simd${P}.c testerutil.c)
+      target_compile_options(${T} PRIVATE ${FLAGS_ENABLE_${SIMD}})
+      target_compile_definitions(${T} PRIVATE ENABLE_${SIMD}=1 USEMPFR=1)
+      set_target_properties(${T} PROPERTIES C_STANDARD 99)
+      target_link_libraries(${T} ${TARGET_LIBSLEEF} ${EXTRA_LIBS})
+      add_dependencies(${T} ${TARGET_HEADERS})
+      add_dependencies(${T} ${TARGET_LIBSLEEF})
+    endforeach()
+
   endif(COMPILER_SUPPORTS_${SIMD})
 endmacro(test_extension)
 
 foreach(SIMD ${SLEEF_SUPPORTED_EXTENSIONS})
   test_extension(${SIMD})
 endforeach()
 
+# Build tester2 scalar
+foreach(P dp sp)
+  set(T "tester2${P}")
+  add_executable(${T} tester2${P}.c testerutil.c)
+  target_compile_definitions(${T} PRIVATE ENABLE_${SIMD}=1 USEMPFR=1)
```

Comment:
I believe `ENABLE_${SIMD}=1` is not  needed here.

Reply:
It needs that macro to be defined.

https://github.com/shibatch/sleef/blob/52494e449233821bf1fc2e93937f24fab87b5c91/src/libm-tester/tester2simddp.c#L26-L32

Reply:
But isn't this the scalar test? You are using tester2sp.c and tester2dp.c, not the simd version.

Reply:
Okay, I see.